README.md

```diff
@@ -1889,65 +1889,17 @@ the Supervisor on a device is outdated and is causing an issue. Usually the best
 way to achieve this is via a balenaOS update, either from the dashboard or via
 the command line on the device.
 
-Should a customer not wish to update the balenaOS version for some reason (or
-should there be an issue where a Supervisor release isn't yet in a new OS
-release but is required on a device), then this can be achieved by running
-the `update-resin-supervisor` script. Here's an example where the version
-(v9.2.2) on balenaOS v2.38.0+rev1 is updated to v10.2.2:
+If updating balenaOS is not desirable or somehow the Supervisor does not update after a host app update this can be resolved by using the nodeJS SDK's [device.setSupervisorRelease](https://www.balena.io/docs/reference/sdk/node-sdk/#devicesetsupervisorreleaseuuidorid-supervisorversionorid-%E2%87%92-codepromisecode). This method contains varies checks to ensure that the update is performed in the safest manner.
```

Comment:
```suggestion
If updating balenaOS is not desirable or a user prefers updating the supervisor independently, this can be resolved by using the Node.js SDK method [device.setSupervisorRelease](https://www.balena.io/docs/reference/sdk/node-sdk/#devicesetsupervisorreleaseuuidorid-supervisorversionorid-%E2%87%92-codepromisecode). This method contains various checks to ensure that the update is performed in the safest manner.
```